src/pages/Login/LoginButton.jsx

```diff
@@ -32,7 +32,9 @@ import styles from './LoginButton.module.css';
  * which can be changed from Google Developer Console later.
  */
 
+// const clientId = process.env.REACT_APP_CLIENT_ID;
 const clientId = process.env.REACT_APP_CLIENT_ID;
+console.log(clientId)
```

Comment:
Probably better to remove commented out codes and console log codes as they were used for debugging purposes.

```suggestion
const clientId = process.env.REACT_APP_CLIENT_ID;
```